src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0; netstandard2.1</TargetFrameworks>
```

Comment:
Do we also want `netcoreapp2.1`, so we get the perf improvements there? Or is getting them only on .NET Core 3.0+ ok?

Reply:
I don't mind adding the ncapp2.1 TFM if there are no concerns with doing so.